ONSdigital/ras-rm-secure-message-v2
# RAS-1046 Migrate Secure Message v2 from GCR to GAR

# What and why?
Update GHA and helm chart to migrate secure message v2 to GAR 
# How to test?
Make sure that the new image is built and pushed to the ci project's GAR 
I can't seem to get spinnaker to pick up the new helm chart at all so you can either
- deploy this pr to your env (which will fail) then go to k8s and edit the deployment with the new image location
   - swap out `eu.gcr.io/ons-rasrmbs-management` with `europe-west2-docker.pkg.dev/ons-ci-rmrasbs/images`
- Or deploy it using helm install and updating the `env`, `namespace`, and the `database.secrets.nameKey` to `db-name` in the `values.yaml`
# Jira
https://jira.ons.gov.uk/browse/RAS-1046
